packages/variable-editor/src/components/variables/data/validation-utils.test.ts

```diff
@@ -1,35 +1,29 @@
+import type { ValidationResult } from '@axonivy/variable-editor-protocol';
 import type { Row } from '@tanstack/react-table';
 import { setupRow, setupVariable } from './test-utils/setup';
-import {
-  containsError,
-  containsWarning,
-  toValidationMessageVariant,
-  validateName,
-  validateNamespace,
-  validationMessagesOfRow
-} from './validation-utils';
+import { containsError, containsWarning, validateName, validateNamespace, validationMessagesOfRow } from './validation-utils';
 import type { Variable } from './variable';
 
-const validationMessages = [
+const validationMessages: Array<ValidationResult> = [
   {
     message: 'message0',
     path: 'Variables.key0.key0',
-    severity: 0
+    severity: 'INFO'
   },
   {
     message: 'message1',
     path: 'Variables.key0.key1',
-    severity: 0
+    severity: 'INFO'
   },
   {
     message: 'message2',
     path: 'Variables.key0.key1',
-    severity: 0
+    severity: 'INFO'
```

Comment:
I assume this was because of the missing gson setting with enums. Did you change this too?